### PR TITLE
Add support for no-name assets, QoL changes

### PR DESCRIPTION
**Pull request by samuelWilliams99**

### CODE REVIEW

File: src/FakePAB/PreBalance.hs [hunk 1]
```diff
@@ -164,7 +174,7 @@ filterNonAda :: Value -> Value
 filterNonAda =
   mconcat
     . map unflattenValue
-    . filter (\(curSymbol, tokenName, _) -> curSymbol /= Ada.adaSymbol && tokenName /= Ada.adaToken)
+    . filter (\(curSymbol, tokenName, _) -> curSymbol /= Ada.adaSymbol || tokenName /= Ada.adaToken)
```
Comment (gege251) [inline src/FakePAB/PreBalance.hs:177]: [nits] I think the second half of this predicate is actually unnecessary, the currency symbol will decide if an asset is ada or not.

**Review by gege251 — APPROVED** [review]

File: src/FakePAB/PreBalance.hs (hunk 1 above)
Comment (samuelWilliams99) [reply src/FakePAB/PreBalance.hs:177]: Very true

Comment (samuelWilliams99) [reply src/FakePAB/PreBalance.hs]: Simplified this and other ada checks, actually allowed me to simplify this function down to a simple key filter

**Review by gege251 — APPROVED** [review]

**Comment by gege251** [comment]
I think you forgot to import the data constructor for Value, otherwise it seems good. Thank you!

**Comment by samuelWilliams99** [comment]
Yeah I totally forgot to just run this PR, turns out had other issues, fixed now :)

**Pull request merged**